app/views/step_by_step_pages/publish_or_delete.html.erb

```diff
@@ -23,43 +23,53 @@
 
-<div class="row publish-or-delete">
-  <div class="col-md-12">
+<div class="govuk-grid-row publish-or-delete">
+  <div class="govuk-grid-column-full">
     <% if @step_by_step_page.has_draft? %>
-      <div class="panel panel-default">
+      <div class="govuk-panel panel-default">
         <div class="panel-body">
-          <p>Before publishing check for broken links.</p>
-          <p>After publishing add this step by step to a mainstream browse category and a taxonomy topic.</p>
-          <%= link_to 'Publish changes', step_by_step_page_publish_path(@step_by_step_page), class: "btn btn-primary publish-button" %>
+          <p class="govuk-body">Before publishing check for broken links.</p>
+          <p class="govuk-body">After publishing add this step by step to a mainstream browse category and a taxonomy topic.</p>
+          <%= render "govuk_publishing_components/components/button", {
+            text: "Publish changes",
+            href: step_by_step_page_publish_path(@step_by_step_page)
+          } %>
           <% if user_can_schedule? %>
-            <p>Or</p>
+            <p class="govuk-body govuk-!-margin-top-5">Or</p>
             <% if @step_by_step_page.scheduled_for_publishing? %>
-              <div class="alert alert-info">
+              <p class="govuk-body">
                 Scheduled to be published at <%= @step_by_step_page.scheduled_at %>
-              </div>
+              </p>
               <%= form_tag step_by_step_page_unschedule_path(@step_by_step_page), method: :unschedule do %>
-                <button class="btn btn-danger">Unschedule publishing</button>
+                <%= render "govuk_publishing_components/components/button", {
+                  text: "Unschedule publishing",
+                  destructive: true
+                } %>
               <% end %>
             <% else %>
-              <%= link_to 'Schedule publish', step_by_step_page_schedule_path(@step_by_step_page), class: "btn btn-info" %>
+              <%= render "govuk_publishing_components/components/button", {
+                text: "Schedule publish",
+                href: step_by_step_page_schedule_path(@step_by_step_page),
+                secondary: true
+              } %>
             <% end %>
           <% end %>
 
           <% if @step_by_step_page.unpublished_changes? %>
-            <p>Or</p>
-            <%= button_to 'Discard changes', { action: "revert", step_by_step_page_id: @step_by_step_page.id }, data: { confirm: 'This will delete your draft. Do you want to discard your changes?' }, class: "btn btn-danger" %>
+            <p class="govuk-body govuk-!-margin-top-5">Or</p>
+            <%= button_to 'Discard changes', { action: "revert", step_by_step_page_id: @step_by_step_page.id }, data: { confirm: 'This will delete your draft. Do you want to discard your changes?' }, class: "gem-c-button govuk-button govuk-button--secondary" %>
```

Comment:
I think this should be a "destructive" button, more so than cancelling scheduling.

Reply:
Why is this a `button_to` and not a button component?